test/integration/setup.js

```diff
@@ -219,5 +220,36 @@ const withClosedForm = (f) => async (service) => {
   return f(service);
 };
 
-module.exports = { testService, testServiceFullTrx, testContainer, testContainerFullTrx, testTask, withClosedForm };
+const createConflict = async (user, container) => {
```

Comment:
It feels weird to me to put this here in setup.js, which I think of as super generic test infrastructure of how tests are run. (I suppose `withClosedForm` is in there....) 

Is there some other place we can put complex scenarios like this for use in multiple tests across multiple files? Could that be a separate issue or PR where we factor out the other places we have duplicate complicated entity logic?